### PR TITLE
new website

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
A work in progress to design the new website

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "sphinx.ext.linkcode",
     "sphinx.ext.autodoc.typehints",
     "sphinx_gallery.gen_gallery",
+    "sphinx_favicon",
```
Comment (GaelVaroquaux) [inline doc/conf.py:61]: Do we need this extension? A favicon can be easily added in the jinja theme. One less extension is one less problem.

Comment (jovan-stojanovic) [reply doc/conf.py:61]: True, we should remove it.

File: build_tools/circle/build_doc.sh [hunk 2]
```diff
@@ -123,6 +123,7 @@ conda create -n $CONDA_ENV_NAME --yes --quiet python="${PYTHON_VERSION:-*}" \
 source activate testenv
 pip install sphinx-gallery
 pip install sphinxext-opengraph
```
Comment (GaelVaroquaux) [inline build_tools/circle/build_doc.sh:125]: Indeed, clearly the setup.cfg extra-requires doc section is not used.

Maybe we should modify the present script to use it?

Comment (GaelVaroquaux) [reply build_tools/circle/build_doc.sh]: Something like
```
pip install -e .[doc]
```
should work, and could allow replacing the list above

**Comment by GaelVaroquaux** [comment]
Let's try to get this building ASAP, and merge it, so as to have something to build upon in an iterative way

**Comment by GaelVaroquaux** [comment]
Merging as the website is rendering. We can improve from this

**Pull request merged**